x-pack/plugins/apm/server/lib/transactions/get_transaction/index.ts

```diff
@@ -60,3 +66,26 @@ export async function getTransaction(
   const resp = await client<Transaction>('search', params);
   return oc(resp).hits.hits[0]._source();
 }
+
+export async function getTransactionWithErrorCount(
+  transactionId: string,
+  traceId: string | undefined,
```

Comment:
If this is targeted for 7.0 `trace.id` is required

Reply:
Actually if this go into master it's by definition targeted for 7.0 so `trace.id` should be required.